### DISCUSSION TITLE
Dragging a VI into another VI's icon replaces the icon

### DISCUSSION THREAD

**Discussion by chrisb2244** [ni/labview-icon-editor#166] [General]
Replacing the icon may be a useful tool, but adding a separate layer with the same content would be more useful in several workflows:

- Copy the whole icon, but then delete various parts to show underlying parts of the previous icon
- Change your mind and just delete the new layer (the change is currently not undoable either - I had to quit the Project without saving anything to return it to the previous logo in a quick test now, although other workflows may also work)
- Reorder it below partial icons, etc

**Comment by CaseyZS** [comment]
What should happen if the dragged VI's icon contains several layers? Merge and copy as a single layer? Copy all layers?

The ability to copy all layers seems more helpful to me.

**Comment by RobustoSystems** [comment]
The ability to copy an individual layer from one VI's icon to another would also be useful.
